src/products/android-l10n/removing_projects.md

```diff
@@ -0,0 +1,38 @@
+# Removing projects from android-l10n
+
+Below are listed the steps to follow in order to safely remove projects from [android-l10n](https://github.com/mozilla-l10n/android-l10n).
+
+## Contact Release Engineering
+
+The first thing to do is to request from the Release Engineering team (“RelEng”) that they disable automation associated to the [android-l10n project](https://github.com/mozilla-l10n/android-l10n) you want to remove.
+Currently, the way to make this request is to send an email directly to RelEng (“Aki” and “Mihai”).
```

Comment:
May be best to not mention specific people by name.

Reply:
Went by their alias on Slack, which turn out to be their first name, but deliberately left out any last names. Will keep them up there for now as it will help me remember who to reach out to next time :) But we can definitely take those out later if you think it's better